<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PainterWidget" name="drawArea" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="X1Label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Y1Label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lineButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polygonButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pointButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteObjectButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="objectTableWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="pointListWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Points</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="translateYvalue">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90000</number>
    </property>
    <property name="maximum">
     <number>90000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="translateX">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="xFactorLabel">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X Factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="translateY">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="translateXvalue">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90000</number>
    </property>
    <property name="maximum">
     <number>90000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="translateLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation (Provide new initial coordinates)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xFactorValue">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>280</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="rotateValue">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>15</number>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yFactorValue">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>280</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="yFactorLabel">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Factor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translateButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>280</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotateLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation (Degrees)</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotateXLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X pivot</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rotateYValue">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="rotateYLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y pivot</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rotateXValue">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PainterWidget</class>
   <extends>QWidget</extends>
   <header>painter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>20</number>
  </property>
  <property name="gridDeltaY">
   <number>20</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
